roar:
  local_planner_manager_node:
    ros__parameters:
      debug: true
      manager_rate: 0.1
      use_sim_time: true
      min_dist: 1.0
lifecycle_manager_local_planning:
  ros__parameters:
    autostart: true
    use_sim_time: true
    node_names: ["/roar/local_planner_manager_node"]
